### PR TITLE
Feature: add `editorBracketHighlight` and ANSI to classic.

### PR DESCRIPTION
**Pull request by yamatsum**
With today's release, the parentheses are now colored. 
https://code.visualstudio.com/updates/v1_60#_high-performance-bracket-pair-colorization
The standard colors don't match the theme, so fix it.

By the way, the classic terminal color was not set, so I set it

before
CleanShot 2021-09-03 at 16 28 12@2x

after
CleanShot 2021-09-03 at 16 27 35@2x

### CODE REVIEW

File: src/classic/theme.js [hunk 1]
```diff
@@ -178,7 +178,33 @@ function getTheme({ style, name }) {
       "panelInput.border": pick({ light: primer.gray[2], dark: primer.gray[1] }),
 
       "terminal.foreground": primer.gray[6],
+      "terminal.background": primer.gray[0],
```
Comment (dominicegginton) [inline src/classic/theme.js:181]: I don't think we need to set the terminal background color. But the rest of your changes LGTM 👍

**Review by dominicegginton — CHANGES_REQUESTED** [review]

File: src/classic/theme.js (hunk 1 above)
Comment (yamatsum) [reply src/classic/theme.js]: thx! Removed background color setting!

**Review by dominicegginton — APPROVED** [review]
LGTM 👍🏻

**Comment by yamatsum** [comment]
Can you merge?

**Comment by dominicegginton** [comment]
@yamatsum will need somone with permissions to merge, might be a little delay as I know there is focus on migration to the new primer version and the addition of the new theme.

**Comment by simurai** [comment]
> By the way, the classic terminal color was not set, so I set it

@yamatsum Thanks for the PR. 🙇 

I tested the terminal colors by running the following commands:

```
echo -e "\033[0mNC (No color)"
echo -e "\033[1;37mWHITE\t\033[0;30mBLACK"
echo -e "\033[0;34mBLUE\t\033[1;34mLIGHT_BLUE"
echo -e "\033[0;32mGREEN\t\033[1;32mLIGHT_GREEN"
echo -e "\033[0;36mCYAN\t\033[1;36mLIGHT_CYAN"
echo -e "\033[0;31mRED\t\033[1;31mLIGHT_RED"
echo -e "\033[0;35mPURPLE\t\033[1;35mLIGHT_PURPLE"
echo -e "\033[0;33mYELLOW\t\033[1;33mLIGHT_YELLOW"
echo -e "\033[1;30mGRAY\t\033[0;37mLIGHT_GRAY"
```

Light | Dark
--- | ---
Screen Shot 2021-09-07 at 10 30 40 | Screen Shot 2021-09-07 at 10 28 52

Most of the dark theme looks great, but we might wanna improve the `WHITE` and `LIGHT_GRAY` colors to make them more visible. Also for the light theme some colors could have more contrast.

I remember this being already an issue with the "default" themes: https://github.com/primer/github-vscode-theme/issues/155. I can try to make these changes... soon-ish.

**Comment by jeffgennari** [comment]
As long as you're updating the theme variables, what do you think about making things a bit more immersive by adding color to the UI. I'm thinking the debugConsole, editorSuggestWidget, debugTokenExpression, and symbolIcon specifically. Perhaps something like the following: 
 
```
 "editorSuggestWidget.focusHighlightForeground": themes({ light: scale.blue[6], dark: scale.blue[3], dimmed: scale.blue[4] }),
      "editorSuggestWidget.highlightForeground": themes({ light: scale.blue[6], dark: scale.blue[3], dimmed: scale.blue[4] }),
      "editorSuggestWidget.selectedForeground": themes({ light: scale.gray[6], dark: scale.gray[3], dimmed: scale.gray[4] }),

      "debugConsole.infoForeground": themes({ light: scale.gray[6], dark: scale.gray[3], dimmed: scale.gray[4] }),
      "debugConsole.warningForeground": themes({ light: scale.yellow[6], dark: scale.yellow[3], dimmed: scale.yellow[4] }),
      "debugConsole.errorForeground": themes({ light: scale.red[6], dark: scale.red[3], dimmed: scale.red[4] }),
      "debugConsole.sourceForeground": themes({ light: scale.purple[6], dark: scale.purple[3], dimmed: scale.purple[4] }),
      "debugConsoleInputIcon.foreground": themes({ light: scale.purple[6], dark: scale.purple[3], dimmed: scale.purple[4] }),

      "debugTokenExpression.name": themes({ light: scale.purple[6], dark: scale.purple[2], dimmed: scale.purple[4] }),
      "debugTokenExpression.value": themes({ light: scale.blue[6], dark: scale.blue[2], dimmed: scale.blue[4] }),
      "debugTokenExpression.string": themes({ light: scale.blue[8], dark: scale.blue[1], dimmed: scale.blue[2] }),
      "debugTokenExpression.boolean": themes({ light: scale.green[6], dark: scale.green[2], dimmed: scale.green[4] }),
      "debugTokenExpression.number": themes({ light: scale.green[6], dark: scale.green[2], dimmed: scale.green[4] }),
      "debugTokenExpression.error": themes({ light: scale.red[6], dark: scale.red[2], dimmed: scale.red[4] }),

      "symbolIcon.arrayForeground": themes({ light: scale.orange[6], dark: scale.orange[3], dimmed: scale.orange[4] }),
      "symbolIcon.booleanForeground": themes({ light: scale.blue[6], dark: scale.blue[3], dimmed: scale.blue[4] }),
      "symbolIcon.classForeground":  themes({ light: scale.orange[6], dark: scale.orange[3], dimmed: scale.orange[4] }),
      "symbolIcon.colorForeground": themes({ light: scale.blue[8], dark: scale.blue[2], dimmed: scale.blue[2] }),
      "symbolIcon.constructorForeground": themes({ light: scale.blue[8], dark: scale.blue[2], dimmed: scale.blue[2] }),
      "symbolIcon.enumeratorForeground": themes({ light: scale.green[6], dark: scale.green[3], dimmed: scale.green[4] }),
      "symbolIcon.enumeratorMemberForeground": themes({ light: scale.green[6], dark: scale.green[3], dimmed: scale.green[4] }),
      "symbolIcon.eventForeground": themes({ light: scale.gray[6], dark: scale.gray[4], dimmed: scale.gray[4] }),
      "symbolIcon.fieldForeground":  themes({ light: scale.orange[6], dark: scale.orange[3], dimmed: scale.orange[4] }),
      "symbolIcon.fileForeground": themes({ light: scale.yellow[6], dark: scale.yellow[3], dimmed: scale.yellow[4] }),
      "symbolIcon.folderForeground": themes({ light: scale.yellow[6], dark: scale.yellow[3], dimmed: scale.yellow[4] }),
      "symbolIcon.functionForeground": themes({ light: scale.purple[6], dark: scale.purple[3], dimmed: scale.purple[4] }),
      "symbolIcon.interfaceForeground":  themes({ light: scale.orange[6], dark: scale.orange[3], dimmed: scale.orange[4] }),
      "symbolIcon.keyForeground": themes({ light: scale.blue[6], dark: scale.blue[3], dimmed: scale.blue[4] }),
      "symbolIcon.keywordForeground": themes({ light: scale.purple[6], dark: scale.purple[3], dimmed: scale.purple[4] }),
      "symbolIcon.methodForeground": themes({ light: scale.purple[6], dark: scale.purple[3], dimmed: scale.purple[4] }),
      "symbolIcon.moduleForeground": themes({ light: scale.red[6], dark: scale.red[3], dimmed: scale.red[4] }),
      "symbolIcon.namespaceForeground": themes({ light: scale.red[6], dark: scale.red[3], dimmed: scale.red[4] }),
      "symbolIcon.nullForeground": themes({ light: scale.blue[6], dark: scale.blue[3], dimmed: scale.blue[4] }),
      "symbolIcon.numberForeground": themes({ light: scale.green[6], dark: scale.green[3], dimmed: scale.green[4] }),
      "symbolIcon.objectForeground": themes({ light: scale.red[6], dark: scale.red[3], dimmed: scale.red[4] }),
      "symbolIcon.operatorForeground": themes({ light: scale.blue[8], dark: scale.blue[2], dimmed: scale.blue[2] }),
      "symbolIcon.packageForeground": themes({ light: scale.purple[6], dark: scale.purple[3], dimmed: scale.purple[4] }),
      "symbolIcon.propertyForeground":  themes({ light: scale.orange[6], dark: scale.orange[3], dimmed: scale.orange[4] }),
      "symbolIcon.referenceForeground": themes({ light: scale.blue[6], dark: scale.blue[3], dimmed: scale.blue[4] }),
      "symbolIcon.snippetForeground": themes({ light: scale.blue[6], dark: scale.blue[3], dimmed: scale.blue[4] }),
      "symbolIcon.stringForeground": themes({ light: scale.blue[8], dark: scale.blue[2], dimmed: scale.blue[2] }),
      "symbolIcon.structForeground": themes({ light: scale.purple[6], dark: scale.purple[3], dimmed: scale.purple[4] }),
      "symbolIcon.textForeground": themes({ light: scale.blue[6], dark: scale.blue[3], dimmed: scale.blue[4] }),
      "symbolIcon.typeParameterForeground": themes({ light: scale.blue[8], dark: scale.blue[2], dimmed: scale.blue[2] }),
      "symbolIcon.unitForeground": themes({ light: scale.blue[6], dark: scale.blue[3], dimmed: scale.blue[4] }),
      "symbolIcon.variableForeground": themes({ light: scale.orange[6], dark: scale.orange[3], dimmed: scale.orange[4] }),
      "symbolIcon.constantForeground": themes({ light: scale.green[6], dark: scale.green[3], dimmed: scale.green[4] }),
```

**Comment by yamatsum** [comment]
@simurai  I tried to fix it, but what about?
CleanShot 2021-09-07 at 21 07 28@2x
CleanShot 2021-09-07 at 21 07 56@2x

**Comment by yamatsum** [comment]
@jeffgennari 
Great idea!
However, since there are a lot of things to change, I would like to make another PR.

**Pull request merged**